tags/3.0/_index.md

```diff
@@ -9,7 +9,7 @@ for manipulating XML documents, internationalization tags, and SQL tags. It also
 existing custom tags with Jakarta Standard Tag Library tags.
 
 * [Jakarta Standard Tag Library 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.jstl/releases/3.0.0)
```

Comment:
@ivargrimstad Wanted to address your comments mentioned here: https://www.eclipse.org/lists/jakarta.ee-spec/msg02435.html

I should remove this `Release Record` above?